src/components/Stats/StatsCollateralChart/StatsCollateral.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import { ResponsiveBar } from '@nivo/bar'
+import { colors } from '@static/theme'
+import { Grid, Typography } from '@material-ui/core'
+import useMediaQuery from '@material-ui/core/useMediaQuery'
+import useStyles from './style'
+
+export const StatsCollateralChart: React.FC = () => {
+  const classes = useStyles()
+
+  interface CoinToChart {
```

Comment:
I think that this definition of interface shold be moved outside definition of the component

Reply:
Done